docs/xdr/features/collect/assets.md

```diff
@@ -172,3 +172,78 @@ This rule creates a new asset for every unseen `host.name`. It attaches the `ipv
 This rule creates a new asset for every unseen `user.name`. It attaches the `user.email` and `user.id` event field values as detection property of the newly created asset.
 
 Questions? Please read our [FAQ](../../FAQ/Assets_qa.md).
+
+# Asset based detections
+
+The Sekoia.io SOC platform now supports the creation of detection patterns for rules or alert-filters that can be customized to apply to specific groups of assets. This feature allows you to enhance the scope of detection by targeting groups of assets that share certain attributes, such as critical servers, or by tuning the detection rules to exclude certain assets, like administrator accounts. This is accomplished by leveraging various asset attributes, as detailed below.
+
+## Asset fields
+
+The following table lists the available fields for defining asset-based detection patterns, along with their types and example values:
+
+| Field                                    | Type                | Example Value                          |
+|------------------------------------------|---------------------|----------------------------------------|
+| `sekoiaio.assets.source.ip.tags`         | Set of Strings      | `{"critical", "internal"}`             |
+| `sekoiaio.assets.source.ip.uuid`         | Set of Strings      | `{NETWORK_UUID, ENTITYSPECIFIC_HOST}`  |
+| `sekoiaio.assets.source.ip.name`         | Set of Strings      | `""`                                   |
+| `sekoiaio.assets.source.ip.criticality_display` | Set of Strings      | `{"high", "low"}`                      |
+| `sekoiaio.assets.source.ip.criticality_value` | Set of Integers    | `{80, 0}`                              |
```

Comment:
`source.ip` could be almost any field so I don't think we should select `source.ip`,  `host.name`, `user.name`, `user.email` and make it seem like this is an exhaustive list

Reply:
good point. Done.